src/style.css

```
@@ -0,0 +1,32 @@
+* {
+  margin: initial;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 15px;
+  width: 15px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 > .stars__star,
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(images/star.svg);
+}
```

You can set default background image for `.stars__star` and then override it in the next selector.
```suggestion
.stars__star {
  height: 15px;
  width: 15px;
  background-repeat: no-repeat;
  background-position: center;
  margin-right: 4px;
  background-image: url(images/star.svg);
}
```